### ISSUE TITLE
Determine daemonize way

### ISSUE THREAD

**Issue by hack3p** [DBMeter/DBMeter#5]
Systemd, supervisor, ..etc or we will be write own daemon? IMHO write a own daemon is not good, because we can a get all daemonize problems how possible (zombie process? or other)

**Comment by anton-okolelov** [comment]
>  IMHO write a own daemon is not good

I agree. It's more simple to daemonize with systemd or something. If we run into problems with it we can change demonization at any time
